crates/sui-graphql-rpc/src/extensions/query_limits_checker.rs

```diff
@@ -214,4 +208,81 @@ impl QueryLimitsChecker {
 
         Ok(())
     }
+
+    fn analyze_selection_set(
+        &self,
+        cfg: &ServiceConfig,
+        fragment_defs: &HashMap<Name, Positioned<FragmentDefinition>>,
+        sel_set: &Positioned<SelectionSet>,
+    ) -> ServerResult<ComponentCost> {
+        // Use BFS to analyze the query and
+        // count the number of nodes and the depth of the query
```

Comment:
> hmm I dont see it
> Mind making a suggestion?

I was referring to how the line break is in a strange place:

```suggestion
        // Use BFS to analyze the query and count the number of nodes 
        // and the depth of the query
```

i.e. very minor, but it just stuck out to my eye.